README.md

```diff
@@ -53,6 +65,15 @@ Type I is used by user libraries, as well as by the web platform in WebIDL and D
 
 Type II is supported if built-in methods create new instances of the subclass. For example, if `Array.prototype.map` or `Array.from` returns instances of subclasses of `Array`. Support for this is provided by delegating to `this.constructor[`@@species`]` inside built-in methods for the default @@species getter.
```

Comment:
Should `@@species` be mentioned here? What is meant by it?

Reply:
should we open an issue around this? cc @syg